src/Symfony/Component/Serializer/Mapping/AttributeMetadataInterface.php

```diff
@@ -59,6 +59,16 @@ public function setSerializedName(string $serializedName = null);
      */
     public function getSerializedName(): ?string;
 
+    /**
+     * Sets the serialization name for this attribute.
+     */
+    public function setSerializedPath(array $serializedPath = null);
+
+    /**
+     * Gets the serialization name for this attribute.
+     */
+    public function getSerializedPath(): ?array;
```

Comment:
dding methods on an interface is a BC break,
these should be added with an `@method` annotation and there should be a line about that in the CHANGELOG of the component + in the root UPGRADE file